homework9/src/siit/hw9/Person.java

```diff
@@ -0,0 +1,37 @@
+package siit.hw9;
+
+import java.time.LocalDate;
+
+public class Person {
+
+    private String first_name;
+    private String last_name;
```

Comment:
Functionally it's correct but it's a better idea to follow Java's naming conventions. In this case the variables' names should have been: `firstName` , `lastName`.